sdk/python/docs/V1TFJobList.md

```diff
@@ -1,12 +1,13 @@
 # V1TFJobList
 
+TFJobList is a list of TFJobs.
 ## Properties
 Name | Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
-**api_version** | **str** | APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources | [optional] 
+**api_version** | **str** | APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources | [optional] 
 **items** | [**list[V1TFJob]**](V1TFJob.md) | List of TFJobs. | 
-**kind** | **str** | Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds | [optional] 
-**metadata** | [**V1ListMeta**](https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/V1ListMeta.md) | Standard list metadata. | [optional] 
+**kind** | **str** | Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds | [optional] 
+**metadata** | [**K8sIoApimachineryPkgApisMetaV1ListMeta**](K8sIoApimachineryPkgApisMetaV1ListMeta.md) |  | [optional] 
```

Comment:
Seems for any core Kubernetes api, they have a prefix like `K8sIoApimachineryPkgApisMeta`?

Reply:
Yes, that's my observation as well. Maybe we could fix that by tweaking the generator?